include/Producer.hpp

```diff
@@ -23,6 +23,8 @@ namespace mediasoupclient
 			  const Producer* producer, webrtc::MediaStreamTrackInterface* newTrack)             = 0;
 			virtual void OnSetMaxSpatialLayer(const Producer* producer, uint8_t maxSpatialLayer) = 0;
 			virtual nlohmann::json OnGetStats(const Producer* producer)                          = 0;
+			virtual void OnPause(Producer* producer) = 0;
```

Comment:
`make format` missing to format C++ code.

Reply:
Unsure about this one.

Reply:
See scripts/tidy.sh